test/unit/materialized_view_address_test.sql

```diff
@@ -0,0 +1,242 @@
+);
+select results_eq(
+  'select physical_address_national_topographical_description from ggircs_swrs.address',
+  ARRAY['A-123-B/456-C-00'::varchar],
+  'ggircs_swrs.address parsed column physical_address_national_topographical_description'
+);
+
+-- Mailing Address Columns
+select results_eq(
+  'select mailing_address_delivery_mode from ggircs_swrs.address',
+  ARRAY['Post Office Box'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_delivery_mode'
+);
+select results_eq(
+  'select mailing_address_po_box_number from ggircs_swrs.address',
+  ARRAY['0000'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_po_box_number'
+);
+select results_eq(
+  'select mailing_address_street_number from ggircs_swrs.address',
+  ARRAY[1234::varchar],
+  'ggircs_swrs.address parsed column mailing_address_street_number'
+);
+select results_eq(
+  'select mailing_address_street_number_suffix from ggircs_swrs.address',
+  ARRAY[null::varchar],
+  'ggircs_swrs.address parsed column mailing_address_street_number_suffix'
+);
+select results_eq(
+  'select mailing_address_street_name from ggircs_swrs.address',
+  ARRAY['00th'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_street_name'
+);
+select results_eq(
+  'select mailing_address_street_type from ggircs_swrs.address',
+  ARRAY['Avenue'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_street_type'
+);
+select results_eq(
+  'select mailing_address_municipality from ggircs_swrs.address',
+  ARRAY['Fort Nelson'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_municipality'
+);
+select results_eq(
+  'select mailing_address_prov_terr_state from ggircs_swrs.address',
+  ARRAY['British Columbia'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_prov_terr_state'
+);
+select results_eq(
+  'select mailing_address_postal_code_zip_code from ggircs_swrs.address',
+  ARRAY['H0H 0H0'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_postal_code_zip_code'
+);
+select results_eq(
+  'select mailing_address_country from ggircs_swrs.address',
+  ARRAY['Canada'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_country'
+);
+select results_eq(
+  'select mailing_address_additional_information from ggircs_swrs.address',
+  ARRAY['The site is located at A-123-B-456-C-789'::varchar],
+  'ggircs_swrs.address parsed column mailing_address_additional_information'
+);
+
+-- Geographic Address columns
+select results_eq(
+  'select geographic_address_latitude from ggircs_swrs.address',
+  ARRAY['23.45125'::varchar],
+  'ggircs_swrs.address parsed column geographic_address_latitude'
+);
+select results_eq(
+  'select geographic_address_longitude from ggircs_swrs.address',
+  ARRAY['-90.59062'::varchar],
+  'ggircs_swrs.address parsed column geographic_address_longitude'
+);
+
+select finish();
+rollback;
+
+-- TODO: Create separate tests for organisation data?
```

Comment:
@wenzowski Should we create extra tests to test the different ways the address can be walked back up the tree? ie tests for when the address context is under facility (done), organisation (not done) and contacts (not done & breaks the view)

Reply:
That would be ideal. One single fixture with all 3 ways should do it